0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -204,29 +204,32 @@
 # In[14]:
 
 
+micdfs = micdfs.loc[:, ~micdfs.columns.str.contains("__antehoc_group1")]
+micdfs.columns = micdfs.columns.str.replace("__antehoc_group0", "", regex=False)
+
+
+# In[15]:
+
+
 agg_merge_cols = ["Metadata_Plate", "Metadata_Well"]
 
 micdfs = pd.merge(
     left=micdfs,
-    right=staindf[agg_merge_cols + ["Metadata_Cell_Count", "Metadata_Condition"]],
+    right=staindf[agg_merge_cols + ["Metadata_Cell_Count"]],
```

Comment:
Is condition no longer important so that is why it is removed?

Reply:
Condition is already present in `micdfs` because of the above change, so this was creating a duplicate condition column